charts/rancher-vsphere-csi/values.yaml

```diff
@@ -145,37 +145,37 @@ global:
 # https://docs.vmware.com/en/VMware-vSphere-Container-Storage-Plug-in/3.0/vmware-vsphere-csp-getting-started/GUID-D4AAD99E-9128-40CE-B89C-AD451DA8379D.html#kubernetes-versions-compatible-with-vsphere-container-storage-plugin-1
 versionOverrides:
   # Versions from https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/release-3.0/manifests/vanilla/vsphere-csi-driver.yaml
-  - constraint: ">= 1.24 < 1.28"
+  - constraint: ">= 1.24 < 1.29"
     values:
       csiController:
         image:
           repository: rancher/mirrored-cloud-provider-vsphere-csi-release-driver
-          tag: v3.0.2
+          tag: v3.1.2
           csiAttacher:
             repository: rancher/mirrored-sig-storage-csi-attacher
-            tag: v4.2.0
+            tag: v4.4.2
           csiResizer:
             repository: rancher/mirrored-sig-storage-csi-resizer
-            tag: v1.7.0
+            tag: v1.9.2
```

Comment:
Same, upsteam has v1.8.0 - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v3.1.2/manifests/vanilla/vsphere-csi-driver.yaml#L264

There are more mismatches in other images so not going to submit more comments on it - probably looking in wrong place or something. 🤷

Reply:
I definitely just copied from master on accident - funny I did the right thing for the CPI chart but not the CSI chart :facepalm: